Action action_chosen
---
int got_action
